src/components/inline-rendered-diff.jsx

```diff
@@ -16,9 +16,13 @@ import SandboxedHtml from './sandboxed-html';
  */
 export default class InlineRenderedDiff extends React.Component {
   render () {
+    let diff = this.props.diff.content.diff;
+    // TODO: remove this switch when new diffs are fully deployed.
+    diff = diff.combined || diff.diff || diff;
```

Comment:
I'm sorry, I know I said I would review this tonight, but when I saw these lines, my eyes started bleeding. This is what I meant when I said weirdness like `this.props.diff.data.content.diff.diff.diff` 😂 

But in all seriousness, I think #161 should be merged first and this rebased before review because `diff.content.diff` soon won't work anymore.

Reply:
I agree. that’s half the reason for edgi-govdata-archiving/web-monitoring-db#186. (This was written before #161 and with the explicit intent to rebase on it whenever it was ready.)